Parsers/ASimProcessEvent/test/SentinelOne_ASimProcessCreate_DataTest.csv

```diff
@@ -0,0 +1,20 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1625 records (32.22%) for field [DvcHostname] of type [Hostname]: [""cent7splunk.ecsp33147.local""] (Schema:ProcessEvent)"
```

Comment:
please split and use only Hostname here. Other values can be put under DvcDomain and DvcFQDN

Reply:
Done